doc/source/manual/xtrabackup_sst.rst

```diff
@@ -343,9 +348,57 @@ The ``tmpdir`` option can be set in the following :file:`my.cnf` groups:
   (if not specified under ``[sst]``)
 * ``[mysqld]`` is used if it is not specified in either of the above
 
+.. option:: wsrep_debug
+
+   :Version: Introduced in 5.7.19-29.22
```

Comment:
1. wsrep_debug variable is present in product for long time now.

2. During one of the past release we introduce wsrep-log-debug. If this variable was set under sst section then we started logging action during SST (applicable only for rsync and XB)

3. Starting this release name of the option is now changed to wsrep-debug to make comply with existing wsrep-debug option.

Reply:
ok